src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnector.java

```diff
@@ -201,7 +201,7 @@ private void closeYBClient() {
     }
 
     private YBClient getYBClientBase(String hostAddress, long adminTimeout, long operationTimeout, long socketReadTimeout,
-                                     int maxNumTablets, String certFile, String clientCert, String clientKey) {
+                                     int maxNumTablets, String certFile, String clientCert, String clientKey, int maxAttempts) {
```

Comment:
If I remember correctly, this method is just used in `YugabyteDBConnector` - we can remove this method now and use the one in `YBClientUtils` as the later one only takes connector configuration as a parameter. We can then use the later method everywhere.